pkg/plugins/resources/remote/store_test.go

```diff
@@ -300,6 +300,39 @@ var _ = Describe("RemoteStore", func() {
 			Expect(err).To(MatchError("(400): some error from the server"))
 		})
 	})
+
+	It("should parse kuma api server error", func() {
+		json := `
+		{
+			"title": "Could not process resource",
+			"details": "Resource is not valid",
+			"causes": [
+				{
+					"field": "path",
+					"message": "cannot be empty"
+				},
+				{
+					"field": "mesh",
+					"message": "cannot be empty"
+				}
+			]
+		}
+		`
+		store := setupErrorStore(json)
+
+		// when
+		meshes := mesh.MeshResourceList{}
+		err := store.List(context.Background(), &meshes)
+
+		// then
+		Expect(err).To(HaveOccurred())
+
+		expectedMsg :=
```

Comment:
I think, it might be better to check if `err` is a structured error (of type `pkg/api-server/types.Error`)